controllers/auth/index.js

```diff
@@ -1,21 +1,128 @@
+/**
+ * @swagger
+ * /register:
+ *   post:
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               fullname:
+ *                 type: string
+ *                 description: The user's name.
+ *                 example: Leanne Graham
+ *               email:
+ *                 type: string
+ *                 description: The user's email.  
+ *                 example: leanne@example.com
+ *               password:
+ *                 type: string
+ *                 description: The user's password.
+ *                 example: password
+ *     summary: Create a new user.
+ *     description: Create a new user.
+ *     tags:
+ *       - Users
+ *     produces:
+ *       - application/json
+ *     consumes:
+ *       - application/json
+ *     parameters:
+ *       - in: body
+ *         name: user
+ *         description: The user to create.
+ *         schema:
+ *           type: object
+ *           required:
+ *             - fullname
+ *             - email
+ *             - password
+ *           properties:
+ *             fullname:
+ *               type: string
+ *             email:
+ *               type: string
+ *             password:
+ *               type: string 
+ *     responses:
+ *       200: 
+ *         description: A JSONPlaceholder user.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 fullname:
+ *                   type: string
+ *                   description: The user's name.
+ *                   example: Leanne Graham
+ *                 email:
+ *                   type: string
+ *                   description: The user's email.
+ *                   example: leanne@example.com   
+ *                 password:
+ *                   type: string
+ *                   description: The user's password.
+ *                   example: password  
+ *       400:
+ *         description: Bad request.
+ *       500:
+ *         description: Internal server error.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 message:
+ *                   type: string
+ *                   description: The error message.
+ *                   example: Internal server error.
+ *       default:
+ *         description: Unexpected error.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 message:
+ *                   type: string
+ *                   description: The error message.
+ *                   example: Unexpected error.
+*/
 
-    return res.json({
-        access_token: '',
-    }).status(200);
-})
+
+router.post("/register", async (req, res) => {
```

Comment:
Check the reviews for `createUser`service, based on that modify this controller as well.